src/core/server/saved_objects/service/lib/filter_utils.test.ts

```diff
@@ -15,6 +15,7 @@ import {
   fieldDefined,
   hasFilterKeyError,
 } from './filter_utils';
+import { nodeBuilder } from '@kbn/es-query';
```

Comment:
I don't think we need another import. `esKuery. nodeBuilder(..)` does the trick. Or let's remove `import * as esKuery from '@kbn/es-query';`